### DISCUSSION TITLE
How to play audio and specify a start time and end time

### DISCUSSION THREAD

**Discussion by xinyuaixiaoyu** [katspaugh/wavesurfer.js#3335] [Q&A]
How to play audio and specify a start time and end time

**Comment by katspaugh** [comment]
Haven't you created a PR to add `stopAt(time)`?

**Reply by xinyuaixiaoyu** [reply]
https://github.com/katspaugh/wavesurfer.js/pull/3325

**Reply by xinyuaixiaoyu** [reply]
When calling the play method of a region, it often fails to play. Can the player provide an API for playing for a period of time?
